tests/unittests/test_handler/test_handler_mounts.py

```diff
@@ -183,6 +185,56 @@ def device_name_to_device(self, path):
 
         return dev
 
+    @mock.patch('cloudinit.util.kernel_version')
+    @mock.patch('cloudinit.util.get_mount_info')
+    def test_swap_creation_method(self, m_kernel_version, m_get_mount_info):
```

Comment:
This looks like it could/should be split into three separate tests (or, perhaps better, a parameterised with 3 cases).

Reply:
According to pytest (https://github.com/pytest-dev/pytest/issues/3484) one cannot decorate a method inside a class that inherits unittest.TestCase (which is this case). I'll split in separate tests and remove the log assertion below and use it just like the `test_swap_integrity()`.